lib/rectangle.js

```diff
@@ -0,0 +1,20 @@
+class Rectangle {
+  area(rect) {
```

Comment:
notice that you're passing in `rect` to each individual method. What if we initialize the rectangle with this and then utilize these properties within each method?

```js
class Rectangle {
  constructor(rectangle) {
    this.height = rectangle.height
    this.width = rectangle.width
  }

  area() {
    return this.height * this.width
  }
}
```

this way you don't have to pass in the argument to every individual method!